packages/ui5-middleware-cfdestination/README.md

```diff
@@ -12,12 +12,12 @@ npm install ui5-middleware-cfdestination --save-dev
 
 - `debug`: `<boolean>`, default: `false`  
 verbose logging
-
 - `port`: `<int>`, default: `5000`  
 port to run the underlying `approuter` on
 
 - `xsappJson`: `<string path>`, default: `"./xs-app.json"`  
 path to the cf-style approuter configuration file `xs-app.json`
+:information_source: the regex for the destination routes must match the pattern : `/[^/]*\/(.*\/)?[^/]*/`
```

Comment:
maybe give one or more examples here to „get“ the meaning of the regex?

Reply:
somehow I'm not so happy with the regex here - I didn't really get why it was filtered by regex instead of the type of the route - maybe we can discuss this separately. But I'm going to add the examples here...